protostar/utils/data_transformer_facade.py

```diff
@@ -15,7 +16,52 @@ class FunctionNotFoundException(BaseException):
     pass
 
 
+class PatchedDataTransformer(DataTransformer):
```

Comment:
Please create a task to make this a  starknet.py PR in this release

Reply:
I'd do it later, once I finish mapping for arguments in events.